jax-controlnet-sprint/README.md

```diff
@@ -49,12 +51,20 @@ For issues with Flax/JAX, Diffusers, Datasets or for questions that are specific
 - JAX: [Issues](https://github.com/google/jax/issues), [Questions](https://github.com/google/jax/discussions)
 - 🤗 Diffusers: [Issues](https://github.com/huggingface/diffusers/issues), [Questions](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)
 - 🤗 Datasets: [Issues](https://github.com/huggingface/datasets/issues), [Questions](https://discuss.huggingface.co/c/datasets/10)
-- Project specific questions: [TODO]()
-- TPU related questions: [TODO]()
+- Project specific questions: Can be asked from each project's own post on #jax-diffusers-ideas channel on Discord.
+- TPU related questions: #jax-diffusers-tpu-support channel on Discord. 
+- General discussion: #jax-diffusers-sprint channel on Discord.
+You will get access to #jax-diffusers-tpu-support and #jax-diffusers-sprint once you are accepted to attend the sprint.
 
-We encourage you to post the link to forum post to the Discord server, instead of directly posting issues or questions. 
+When asking for help, we encourage you to post the link to [forum](https://discuss.huggingface.co) post to the Discord server, instead of directly posting issues or questions. 
 This way, we make sure that the everybody in the community can benefit from your questions, even after the community sprint.
 
-> 💡 Note: If you have signed up on the google form, but you are not in the Discord channel, please leave a message on [(TODO) the official forum announcement] () and ping `@mervenoyan`, `@sayakpaul`, and `@patrickvonplaten`. We might take a day to process these requests.
+> 💡 Note: After 10th of April, if you have signed up on the google form, but you are not in the Discord channel, please leave a message on [the official forum announcement] (https://discuss.huggingface.co/t/controlling-stable-diffusion-with-jax-and-diffusers-using-v4-tpus/35187/2) and ping `@mervenoyan`, `@sayakpaul`, and `@patrickvonplaten`. We might take a day to process these requests.
+
+## Talks
+
+We will have talks from folks working at JAX & TPU teams at Google, diffusers and ethics teams at Hugging Face and awesome open-sorcerers working in generative AI. We will update this post with links to the talks, so keep an eye here or on Discord in diffusion models core-announcements channel and set your reminders!
+
+- [Link to the talks on 17th of April](https://www.youtube.com/watch?v=SOj2sxgvFe0), we will be hosting Andreas Steiner, Margaret Mitchell and Boris Dayma. 
```

Comment:
Maybe let's do a table similar to https://github.com/huggingface/transformers/tree/main/examples/research_projects/jax-projects#talks  (ok if all links go to same video) so we also have the topics

Reply:
Looks neat!